forc-plugins/forc-crypto/src/sha256.rs

```diff
@@ -0,0 +1,8 @@
+use fuel_crypto::Hasher;
+
+/// Hashes a given data to Sha256
+pub fn hash<T: Into<Vec<u8>>>(data: T) -> anyhow::Result<Vec<u8>> {
+    let mut hasher = Hasher::default();
```

Comment:
Same comment as `keccak256.rs` about error handling & input validation.

Reply:
I missed the context here @sdankel. Would you mind giving it again please? Anyway, the error was that on empty first argument `stdin` was read. That is fixed now.